examples/simple/launch.sh

```diff
@@ -62,22 +62,44 @@ AGENTPID="$!"
 # Let the agent start up
 sleep 2
 
-export PCCMD="dbus-send --${BUS} --print-reply --dest=com.pelagicore.SoftwareContainerAgent /com/pelagicore/SoftwareContainerAgent"
+# Destination
+PCNAME="com.pelagicore.SoftwareContainerAgent"
+# Object path
+PCOBJPATH="/com/pelagicore/SoftwareContainerAgent"
+# Prefix for dbus methods
+AGENTPREFIX="com.pelagicore.SoftwareContainerAgent"
+export PCCMD="dbus-send --${BUS} --print-reply --dest=$PCNAME $PCOBJPATH"
 
```

Comment:
This example, and e.g. the simple example, all use "PC" as part of a lot of variable names. This doesn't make any sense anymore. No prefix or "SC" is more suitable.